CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2024-06-09
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/). This release is titled as v20240609. 
+
+### Announcements
+* Starting 1.30 Kubernetes version and 1.27 LTS versions, beta apis will be disabled by default, when you upgrade to [them](https://learn.microsoft.com/azure/aks/upgrade-aks-cluster?tabs=azure-cli#before-you-begin). There will be an option provided to explicitly enable beta apis closer to the 1.30 release.
+
+
+### Release Notes
+
+* Features:
+  * Generally Available - [Security Patch channel - Live patching mechanism](https://learn.microsoft.com/azure/aks/auto-upgrade-node-os-image?tabs=azure-cli) for VHD updates.
+  * AKS Patch version [1.27.13](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.27.md#v12713) is now available. 
+
+* Preview Features:
+  * AKS version [1.30](https://kubernetes.io/blog/2024/04/17/kubernetes-v1-30-release/) is available in preview. 
+
+* Behavioral Changes:
+  * asm-1-19 is no longer supported. If this revision is in use, please upgrade for continued support. More information about mesh upgrades and version support can be found [here](https://aka.ms/asm-aks-upgrade-docs).
+
+* Bug Fixes:
+  * Updating coredns to use image [v1.9.4-hotfix.20240520](https://github.com/aks-lts/coredns/tree/release-1.9) on all AKS clusters with version 1.24+. We still use coredns build version v1.9.4, we have only fixed the [CVE vulnerabilities](https://github.com/aks-lts/coredns/commit/ba698d28c2ab8d9db0951592be631885e4134e5) on this.
+  * Replacing cilium 1.14.4 with new patch 1.14.10 for k8s 1.29+ fixes the [issue](https://github.com/cilium/cilium/issues/18706).
```

Comment:
Replacing cilium 1.14.4 with new patch 1.14.10 for k8s 1.29+ fixes the [issue](https://github.com/cilium/cilium/issues/18706). 

to 

Cilium has been upgraded to version 1.14.10 for K8s version 1.29+. This patch upgrade fixes a bug where [the host network is broken and remains broken even if the underlying interface goes up again](https://github.com/cilium/cilium/issues/18706).